Color_Count.c

```diff
@@ -0,0 +1,13 @@
+const char* MajorColorNames[] = {
```

Comment:
the file name Color_count.c indicates that there is more with counting of colors. whereas this file is only about defining colors and the number of minor and major colors. It can be renamed to a different name, ex. Colors.c